src/components/SEO/SEO.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { useStaticQuery, graphql } from 'gatsby';
+import PropTypes from 'prop-types';
+
+const SEO = ({ postData, frontmatter = {}, postImage, isBlogPost }) => {
+  const { site } = useStaticQuery(
+    graphql`
+      {
+        site {
+          siteMetadata {
+            title
+            description
+            siteUrl
+            image
+            author {
+              name
+            }
+            organization {
+              name
+              url
+              logo
+            }
+            social {
+              twitter
+              fbAppID
+            }
+          }
+        }
+      }
+    `,
+  );
+
+  const { siteMetadata: seo } = site;
+
+  const postMeta =
+    frontmatter || postData.childMarkdownRemark.frontmatter || {};
+
+  const title = postMeta?.title || seo?.title;
```

Comment:
Let's jut leverage the `seo` data for now that's coming from the `gatsby-config` If / when we get posts we'll write the logic out for that!

This can be done for lines 39-42